### DISCUSSION TITLE
Объединение таблиц по типу join

### DISCUSSION THREAD

**Discussion by Makhmut2445** [totumonline/totum-ru-issues-and-discussions#106] [Есть идея, а что если...]
Есть возможность объединения таблиц в одну?
Допустим есть таблица Title1, поля name1, name2,name3
Так же есть таблица  Title2, поля name1, name4,name3
Объединеная будет такая:
 Title3, поля name1, name2,name4,name3.
При этом записи в полях с одинаковыми названиями полей становятся в одно поле...

**Comment by ttm-alexeypolunin** [comment]
То, что вы описали — это `join`. Он является элементом обработки данных, а таблицы в тотум — это элемент хранения.

Более того, таблицы имеют жесткую структуру. Поэтому основная архитектура решения должна быть `joinless`.

Сам `join` как функцию мы добавим в MIT, иногда это может быть полезно. Дополнительно к ней в PRO будет виртуальный тип таблиц с не жесткой структурой не предназначенных для хранения (типа временных) предназначенных для отчетов и промежуточной обработки.
